package.json

```diff
@@ -64,9 +64,9 @@
   },
   "bin": "build/app.js",
   "engines": {
-    "node": ">=12 <13"
+    "node": ">=14 <15"
```

Comment:
Node 15 reaches EOL on June 1st, so we might as well guard against it right away.

Reply:
Hm, maybe this is dumb since the code can also be used as a plugin, not just a Docker image. Should it be limited to any 12.x.x or 14.x.x?